File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number)
    text = ""
    if number % 3 == 0
        text *= "Pling"
    end
    if number % 5 == 0
        text *= "Plang"
    end
    if number % 7 == 0
        text *= "Plong"
    end
    if text == ""
        text *= "$number"
    end
    text
end

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "fmt"

func Convert(number int) string {
	text := ""
    if number % 3 == 0 {
        text += "Pling"
    }
    if number % 5 == 0 {
        text += "Plang"
    }
    if number % 7 == 0 {
        text += "Plong"
    }
    if text == "" {
        return fmt.Sprintf("%d", number)
    } else {
        return text
    }
}
